src/server/db/inmemory.js

```diff
@@ -44,6 +44,19 @@ export class InMemory {
     return await this.games.get(id);
   }
 
+  /**
+   * Remove the game state from the in-memory object.
+   * @param {string} id - The game id.
+   * @returns {object} - The removed game state, or undefined
+   *                     if no game was found with this id.
+   */
+  async remove(id) {
```

Comment:
All the DB implementations need to have the same signature.  You've implemented `remove` and `list` for the in-memory implementation, but someone could be using mongo and that code would fail.

At the same time, I don't want to put too much burden on contributors to have to update multiple DB implementations every time something changes.  I'm happy to bring the Firebase and Mongo implementations in line with this by adding changes to your PR (won't have time till next week, though).  In the meantime, feel free to get to it sooner if you have the time.